pgmq.go

```diff
@@ -145,9 +161,32 @@ func (p *PGMQ) SendBatch(ctx context.Context, queue string, msgs []json.RawMessa
 
 // SendBatchWithDelay sends a batch of messages to a queue with a delay. The
 // delay is specified in seconds. The message ids, unique to the queue, are
-// returned.
+// returned. Only supported in pgmq-pg17 and above.
```

Comment:
Only supported by pgmq-pg17 should be in the send with timestamp method right?

Reply:
fixed